rohmu/notifier/http.py

```diff
@@ -85,7 +87,7 @@ def background_http_request(
 
 
 def initialize_background_thread(
-    queue: Queue,
+    queue: "Queue[HTTPNotifyJob]",
```

Comment:
Will `from __future__ import annotations` help with not using a string hint here?

Reply:
Probably. I certainly added this before adding the future import. So I will review all these cases after lunch.